storage/database/db_manager.go

```diff
@@ -2038,6 +2054,11 @@ func (dbm *databaseManager) ReadRecentGovernanceIdx(count int) ([]uint64, error)
 			return nil, e
 		}
 
+		// Make sure idxHistory should be in ascending order
+		sort.Slice(idxHistory, func(i, j int) bool {
+			return idxHistory[i] < idxHistory[j]
+		})
```

Comment:
If sort works here, that means the code in line#2016~line#2020 didn't work.
Is this code needed for handling (already) inserted values?

Reply:
@ehnuje You are right. `idxHistory` should be sorted already in the database. It is a kind of defensive programming to make sure the order. If `idxHistory` is corrupted, the Klaytn node could experience a bad block and hard to failback again. Indeed, the function will be called frequently. 
If you still think the code is not necessary, I will consider removing it.

Reply:
@aidan-kwon I think it's okay. However, why does this bug happen on KES environment?

Reply:
@ehnuje 
1. When `ken init` is executed, `WriteGovernance` is called **without loading of index cache** `governance.idxCache` and `WriteGovernance` finally calls `WriteGovernanceIdx`.

https://github.com/klaytn/klaytn/blob/3281053dcbfab19821f32dbfcfdf03b1a9dc3d34/cmd/utils/nodecmd/chaincmd.go#L153-L158

2. `WriteGovernanceIdx` doesn't filter an index that is not in ascending order. So, I added the following logic inside of 
`WriteGovernanceIdx` also as `WriteGovernance` do. 
https://github.com/klaytn/klaytn/blob/3281053dcbfab19821f32dbfcfdf03b1a9dc3d34/storage/database/db_manager.go#L2016-L2020